server/cache/resource_cache.go

```diff
@@ -26,16 +27,20 @@ func NewResourceCacheWithTimeout(client kubernetes.Interface, ctx context.Contex
 		cache:                NewLRUTtlCache(timeout, 2000),
 		client:               client,
 		ServiceAccountLister: informerFactory.Core().V1().ServiceAccounts().Lister(),
+		informerFactory:      informerFactory,
 	}
-	informerFactory.Start(ctx.Done())
-	informerFactory.WaitForCacheSync(ctx.Done())
 	return cache
 }
 
 func NewResourceCache(client kubernetes.Interface, ctx context.Context, namespace string) *ResourceCache {
 	return NewResourceCacheWithTimeout(client, ctx, namespace, time.Minute*1)
 }
 
+func (c *ResourceCache) Run() {
```

Comment:
pass `stopCh` (which wil be `ctx.Done()`)

Reply:
done